Tests/Shared.Specs/CollectionAssertionSpecs.cs

```diff
@@ -2337,6 +2336,231 @@ public void When_testing_collections_not_to_be_equivalent_against_same_collectio
 
         #endregion
 
+        #region Contain Equivalent Of
+
+        [Fact]
+        public void When_collection_contains_object_equal_to_another_it_should_succeed()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            int item = 2;
+            IEnumerable collection = new[] { 1, item, 3 };
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Act / Assert
+            //-----------------------------------------------------------------------------------------------------------
+            collection.Should().ContainEquivalentOf(item);
+        }
+
+        [Fact]
+        public void When_collection_contains_object_equivalent_to_another_it_should_succeed()
```

Comment:
🤔 Isn't this test is equivalent to `When_collection_contains_object_equal_to_another_it_should_succeed` as `int` is a valuetype?

Reply:
Absolutely. Changed it to object.